### ISSUE TITLE
UI: Add external link button

### ISSUE THREAD

**Issue by jasmussen** [WordPress/gutenberg#969]
Once you've published your post, you want the link to that post. You can get that link through the permalink interface in #577, and you can get it in the notice you get when you publish in #967. 

But outside of that, should there be another affordance for getting the link? 

Here's a design that we mocked up for this editor, and have since been testing in Calypso:

![external link](https://cloud.githubusercontent.com/assets/1204802/26672670/469402b6-46ba-11e7-8b4b-d18d7dd4ee4a.png)

![external link hover](https://cloud.githubusercontent.com/assets/1204802/26672673/48169bee-46ba-11e7-8acf-bab9aa0381ca.png)

**Comment by afercia** [comment]
Where in the tab sequence this link will be placed? For keyboard users discoverability is fundamental. Also, as noted in other issues too, one specific SCAG requirement is to group controls in a logical way. Related controls should be grouped. To me, the placement of this link looks far from ideal from a document structure perspective.

**Comment by paulwilde** [comment]
The admin toolbar already has a 'View Post/Page' link, which is more obvious than a random icon, so I question whether this is even needed?

**Comment by paaljoachim** [comment]
What about renaming the preview button to just calling it "view" ?

**Comment by aduth** [comment]
"Preview" still has a place for published posts, as an autosave is similar in purpose as a draft equivalent for unsaved changes to published posts. You may have seen this notice:

![Autosave](https://user-images.githubusercontent.com/1779930/27035028-db347734-4f4d-11e7-8750-4372ddd62033.png)

So a user may potentially want both "View" and "Preview" links for a published post if that's the workflow they prefer for comparing the saved post and unsaved (more accurately un-published) changes.

**Comment by joedolson** [comment]
Is there any reason a "View" link couldn't be placed immediately next to the 'Preview' link? That would certainly meet the expectations for proximity of controls, if the link is added at all. I agree that the existing Admin Toolbar link may satisfy this need, and there isn't currently a 'View' link in the editor, so that's not an expectation.

**Comment by jasmussen** [comment]
> Is there any reason a "View" link couldn't be placed immediately next to the 'Preview' link?

The primary reason is keeping a UI that stays largely the same from mobile to desktop, and the more we add to the top bar, the harder it is to fit it on mobile. An option is to simply hide the View link on mobile, but I'd larger we explore other options first.

**Comment by jasmussen** [comment]
Closing for now, as we have a publish notice with a link, and because other discussions on this suggest it might not be necessary.
